package.json

```diff
@@ -61,8 +61,9 @@
     "postinstall": "rm -rf ./packages/loot-design/node_modules/react && rm -rf ./packages/mobile/node_modules/react && rm -rf ./node_modules/react-native && patch-package"
   },
   "devDependencies": {
-    "@babel/plugin-transform-modules-commonjs": "^7.15.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.18.2",
     "cross-env": "^5.1.5",
+    "full-icu": "^1.4.0",
```

Comment:
I'm not sure this is necessary.

Looking at the [readme for full-icu](https://github.com/nodejs/full-icu-npm#full-icu) this is only necessary if you're using a version of Node.js under 13. The oldest LTS version of Node.js is 14 so I think we can safely assume that this is installed by default. Are you running an old version of Node?

Reply:
I can have a play around with this tomorrow to see if it really makes a difference.

Reply:
Ha! I thought I was on v16...usually am. Turns out I downgraded to v12 for Actual dev. Whoops! (I think I somewhere that's the version it targeted, but I don't think that's actually true.)
I'll get this out of here.

Reply:
Thanks for the catch!